core/clingutils/res/TClingUtils.h

```diff
@@ -164,13 +164,15 @@ class TClingLookupHelper : public TClassEdit::TInterpreterLookupHelper {
    TNormalizedCtxt    *fNormalizedCtxt;
    ExistingTypeCheck_t fExistingTypeCheck;
    AutoParse_t         fAutoParse;
+   bool               *fIsShuttingDown;
```

Comment:
Can we disable the calls to the TClingLookupHelper in general rather than propagating this flag?

PS: or have a shut down mode for this particular class. It should be cleaner and equivalent as the interpreter should destroy this maybe first?

Reply:
I will renamed this variable fInterpreterIsShuttingDown.

When the lookup helper goes away (another commit in this PR), it usage is disable.

This data member is used to indicate "some function static variable is going away in some of the routine the interpreter is using, nothing is safe anymore, do less"